webpack.config.js

```diff
@@ -18,26 +24,62 @@ module.exports = {
     extensions: ['', '.js', '.jsx']
   },
   module: {
+    noParse: [
+      Path.join(__dirname, "node_modules/react/dist/react.min.js"),
+      Path.join(__dirname, "node_modules/react-intl/dist/react-intl.min.js"),
+      Path.join(__dirname, "node_modules/react-router/umd/ReactRouter.min.js"),
+      Path.join(__dirname, "node_modules/react-ga/dist/index.min.js"),
+      Path.join(__dirname, "pages/sequential.jsx")
+    ],
     loaders: [
-      { test: /\.js$/, loaders:  ['babel-loader'], exclude: ['node_modules'] },
-      { test: /\.jsx$/, loaders: ['babel-loader'], exclude: ['node_modules'] },
-      { test: /\.json$/, loaders: ['json-loader'], exclude: ['node_modules'] },
+      { test: /\.js$/, loaders:  ['babel-loader'], exclude: exclude },
+      { test: /\.jsx$/, loaders: ['babel-loader'], exclude: exclude },
+      { test: /\.json$/, loaders: ['json-loader'], exclude: exclude },
       { test: /\.less$/, loader: ExtractTextPlugin.extract(
                     'css?sourceMap!less?sourceMap'
-                ), exclude: ['node_modules'] }
+                ), exclude: exclude }
     ],
     preLoaders: [
-      { test: /\.jsx$/, loaders: ['eslint-loader'], exclude: ['node_modules'] }
+      { test: /\.jsx$/, loaders: ['eslint-loader'], exclude: exclude }
     ]
   },
+  resolve: {
+    alias: {
+      "react$": Path.join(__dirname, "node_modules/react/dist/react.min.js"),
+      "react-intl$": Path.join(__dirname, "node_modules/react-intl/dist/react-intl.min.js"),
+      "react-router$": Path.join(__dirname, "node_modules/react-router/umd/ReactRouter.min.js"),
+      "react-ga$": Path.join(__dirname, "node_modules/react-ga/dist/index.min.js")
+    }
+  },
   eslint: {
     emitError: true,
     emitWarning: true
   },
+  devtool: 'inline-eval-cheap-source-map',
```

Comment:
Don't understand this one yet, but it was recommended it. It's perf boost didn't blow me away.